shared-module/displayio/EPaperDisplay.c

```diff
@@ -366,6 +384,38 @@ STATIC bool displayio_epaperdisplay_refresh_area(displayio_epaperdisplay_obj_t *
     return true;
 }
 
+STATIC bool _clean_area(displayio_epaperdisplay_obj_t *self) {
+    uint16_t width = displayio_display_core_get_width(&self->core);
+    uint16_t height = displayio_display_core_get_height(&self->core);
+
+    // Allocated and shared as a uint32_t array so the compiler knows the
+    // alignment everywhere.
+    uint8_t buffer[width / 2];
+    memset(buffer, 0x77, width / 2);
```

Comment:
Did you mean to declare this as `uint32_t`? I though that's what the comment would imply. And you cast it to `uint8_t` below.

Reply:
Removed the comment. It was copied from below.